docs/vcpkg-schema-definitions.schema.json

```diff
@@ -0,0 +1,715 @@
+              "description": "The locale under which this install should be applied."
+            },
+            "nametag": {
+              "type": "string",
+              "description": "A string to add to the name of the artifact when extracted on disk."
+            }
+          },
+          "required": [
+            "unzip"
+          ],
+          "additionalProperties": false
+        },
+        {
+          "properties": {
+            "untar": {
+              "$ref": "#/definitions/uri-or-uris"
+            },
+            "sha256": {
+              "$ref": "#/definitions/sha256"
+            },
+            "sha512": {
+              "$ref": "#/definitions/sha512"
+            },
+            "strip": {
+              "type": "integer",
+              "minimum": 0,
+              "default": 0
+            },
+            "transform": {
+              "type": "string",
+              "description": "An SED expression to run on the file names before extracting them."
+            }
+          },
+          "required": [
+            "untar"
+          ],
+          "additionalProperties": false
+        },
+        {
+          "properties": {
+            "nupkg": {
+              "type": "string",
+              "format": "uri"
+            },
+            "sha256": {
+              "$ref": "#/definitions/sha256"
+            },
+            "sha512": {
+              "$ref": "#/definitions/sha512"
+            },
+            "strip": {
+              "type": "integer",
+              "minimum": 0,
+              "default": 0
+            },
+            "transform": {
+              "type": "string",
+              "description": "An SED expression to run on the file names before extracting them."
+            },
+            "lang": {
+              "type": "string",
+              "description": "The locale under which this install should be applied."
+            },
+            "nametag": {
+              "type": "string",
+              "description": "A string to add to the name of the artifact when extracted on disk."
+            }
+          },
+          "required": [
+            "nupkg"
+          ],
+          "additionalProperties": false
+        },
+        {
+          "properties": {
+            "git": {
+              "type": "string",
+              "description": "The http or ssh remote from which a git repo will be fetched."
+            },
+            "commit": {
+              "type": "string",
+              "description": "The commit SHA or ref that will be checked out."
+            },
+            "options": {
+              "type": "array",
+              "items": {
+                "type": "string",
+                "enum": [
+                  "full",
+                  "recurse"
+                ]
+              }
+            },
+            "subdirectory": {
+              "type": "string",
+              "default": ""
+            }
+          },
+          "required": [
+            "git"
```

Comment:
Is `"commit"` required or does it default to `HEAD`? (if so, it would be nice to add that as `"default": "HEAD"`).

Reply:
I do not know.